### PR TITLE
Turn on the contract actions sidecar in mainnet and testnet

### LINKED ISSUE #5686: Turn on the contract actions sidecar in mainnet and testnet
TIme to try again to turn on the contract actions sidecar in mainnet and testnet.

Already on in previewnet.  Perf issues have been resolved.  Mirror node is ready. No operational issues expected.

For:

[] `develop` - contract sidecars
[x] `release/0.37` - contract sidecars + record stream compression

### PR DESCRIPTION
**Pull request by david-bakin-sl**
**Description**:

Done by _removing_ the override lines from mainnet and testnet specific configurations, as the default `bootstrap.properties` already have all three sidecars on.

**Related issue(s)**:

Fixes #5686 

**Notes for reviewer**:

Ready to go again.

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Comment by Nana-EC** [comment]
@david-bakin-sl let's rebase and also get this into `release/0.37` as we did for 0.36

File: hedera-node/hedera-app/src/test/resources/bootstrap.properties [hunk 1]
```diff
@@ -90,7 +90,7 @@ expiry.throttleResource=expiry-throttle.json
 contracts.redirectTokenCalls=true
 contracts.referenceSlotLifetime=31536000
 contracts.scheduleThrottleMaxGasLimit=5000000
-contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE
+contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE,CONTRACT_ACTION
```
Comment (mhess-swl) [inline hedera-node/hedera-app/src/test/resources/bootstrap.properties:93]: Is this the right file? I would think we'd need to add this line to `hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties`

**Review by Nana-EC — APPROVED** [review]

File: hedera-node/hedera-app/src/test/resources/bootstrap.properties (hunk 1 above)
Comment (Nana-EC) [reply hedera-node/hedera-app/src/test/resources/bootstrap.properties:93]: Should already exist there https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties#L94

**Review by mhess-swl — DISMISSED** [review]

File: hedera-node/hedera-app/src/test/resources/bootstrap.properties (hunk 1 above)
Comment (david-bakin-sl) [reply hedera-node/hedera-app/src/test/resources/bootstrap.properties:93]: It's already there.  To the best of my knowledge there are two kinds of bootstrap files: _Ur_-bootstrap files which contain _every_ property (and _must_ contain _something_ for every property) and _delta_ bootstrap files (my name) which contain overrides.  This is an _ur_-bootstrap file so it needs this property.  AFAIK.  TBH, this is a rather _new_ file with the rather _new_ `hedera-app` - I don't know under what circumstances it is used.

**Review by Nana-EC — APPROVED** [review]

**Review by mhess-swl — APPROVED** [review]

**Pull request merged**